specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations.json

```diff
@@ -0,0 +1,1443 @@
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation",
+      "required": [
+        "analysisInput",
+        "parameters"
+      ]
+    },
+    "CustomConversationTaskParameters": {
+      "type": "object",
+      "description": "Input parameters necessary for a CustomConversation task.",
+      "properties": {
+        "projectName": {
+          "type": "string",
+          "description": "The name of the project to use."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "The name of the deployment to use."
+        },
+        "verbose": {
+          "type": "boolean",
+          "description": "If true, the service will return more detailed information in the response."
+        },
+        "isLoggingEnabled": {
+          "type": "boolean",
+          "description": "If true, the service will keep the query for further review."
+        },
+        "stringIndexType": {
+          "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "directTarget": {
+          "type": "string",
+          "description": "The name of a target project to forward the request to."
+        },
+        "targetProjectParameters": {
+          "type": "object",
+          "description": "A dictionary representing the parameters for each target project.",
+          "additionalProperties": {
+            "$ref": "#/definitions/AnalysisParameters"
+          }
+        }
+      },
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomConversationalTaskResult": {
+      "description": "The results of a CustomConversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTaskResult"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "result": {
+          "$ref": "#/definitions/AnalyzeConversationResult"
+        }
+      },
+      "required": [
+        "result"
+      ],
+      "x-ms-discriminator-value": "CustomConversationResult"
+    },
+    "ConversationItemBase": {
+      "type": "object",
+      "description": "The abstract base for a user input formatted conversation (e.g., Text, Transcript).",
+      "properties": {
+        "id": {
+          "description": "The ID of a conversation item.",
+          "type": "string"
+        },
+        "participantId": {
+          "description": "The participant ID of a conversation item.",
+          "type": "string"
+        },
+        "language": {
+          "description": "The override language of a conversation item in BCP 47 language representation.",
+          "type": "string"
+        },
+        "inputModality": {
```

Comment:
We do not use input as prefix to other parameters, i suggest avoid this renaming.

Reply:
Resolved